packages/dashboard/src/pages/Wallets/components/Transfer/TransferERC20.tsx

```diff
@@ -58,6 +58,10 @@ export const TransferERC20 = memo<TransferERC20Props>(({ token }) => {
     const is1559Supported = useMemo(() => isEIP1559Supported(chainId), [chainId])
     const { gasNow } = useGasOptions()
 
+    useEffect(() => {
+        setToken(token)
```

Comment:
To avoid confusing, rename `setToken` to `setSelectedToken`
```suggestion
        setToken(token)
```